pkg/agent/runtime.go

```diff
@@ -129,6 +143,35 @@ func (m *DebugAttacher) DebugContainer(container, image string, authStr string,
 	return nil
 }
 
+
+func (m *DebugAttacher) SetContainerLxcfs(container string) error {
+	ctx, cancel := m.getContextWithTimeout()
+	defer cancel()
+	containerInstance, err := m.client.ContainerInspect(ctx, container)
+	if err != nil {
+		return err
+	}
+	for _, mount := range containerInstance.Mounts {
+		if mount.Destination == LxcfsRootDir {
+			log.Printf("remount lxcfs when the rootdir of lxcfs of target container has been mounted. \n\t ")
+			for _, procfile := range LxcfsProcFiles {
+				nsenter := &nsenter.MountNSEnter{
+					Target: 	containerInstance.State.Pid,
+					MountLxcfs: true,
+				}
+				stdout, stderr, err := nsenter.Execute("--","mount", "-B", LxcfsHomeDir+procfile, procfile)
+				if err != nil {
+					log.Printf(stderr)
+				}
+				log.Printf(stdout)
```

Comment:
What are these for?

Reply:
it seems that I should append some descriptions.
`				_, stderr, err := nsenter.Execute("--", "mount", "-B", LxcfsHomeDir+procfile, procfile)
				if err != nil {
					log.Printf("bind mount lxcfs files failed. \n\t reason: %s", stderr)
					return err
				}`